client/ayon_core/plugins/publish/extract_thumbnail.py

```diff
@@ -349,7 +349,8 @@ def _get_filtered_repres(self, instance):
                 continue
 
             if "review" not in tags:
```

Comment:
I would add one modification. Collect representations with `"review"` tag into different list and withou the tag into other list and concatenate them at the end -> so they are in "prefered" order.

```
review_repres = []
other_repres = []
for repre in src_repres:
    ...

    has_review_tag = "review" in tags
    if not has_review_tag and not self._is_valid_images_repre(repre):
        continue
    ...

    if has_review_tag:
        review_repres.append(repre)
    else:
        other_repres.append(repre)

return review_repres + other_repres
```